assignment6.txt

```diff
@@ -0,0 +1,14 @@
+Priyanka Musti
+
+1.
+Yes there is no return 0 statement at the bottom of the code, you could fix this by putting ruturn 0 at the bottom of the code. Also, num isnt given a value before it is put in as the array size
```

Comment:
Right! You can fix the problem with the array size by simply moving the array declaration after input is scanned from the user.